<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>快播私有云测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 500px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; }
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class='heading'>
<h1>快播私有云测试报告</h1>
<p class='attribute'><strong>Start Time:</strong> 2014-03-25 01:00:04</p>
<p class='attribute'><strong>Duration:</strong> 0:00:35.390000</p>
<p class='attribute'><strong>Status:</strong> Error 34</p>

<p class='description'>运行浏览器：chrome</p>
</div>



<p id='show_detail_line'>Show
<a href='javascript:showCase(0)'>Summary</a>
<a href='javascript:showCase(1)'>Failed</a>
<a href='javascript:showCase(2)'>All</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>Test Group/Test case</td>
    <td>Count</td>
    <td>Pass</td>
    <td>Fail</td>
    <td>Error</td>
    <td>View</td>
</tr>

<tr class='errorClass'>
    <td>aalogin_sta.Login</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c1',1)">Detail</a></td>
</tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_login: 用户登陆</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        error</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\aalogin_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>addseed_sta.Addsend</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c2',2)">Detail</a></td>
</tr>

<tr id='ft2.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_qvod: 添加qvod种子文件</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        error</a>

    <div id='div_ft2.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\addseed_sta.py", line 20, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_seed: 添加快播链接</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.2')" >
        error</a>

    <div id='div_ft2.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\addseed_sta.py", line 20, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>collect_sta.Collect</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c3',1)">Detail</a></td>
</tr>

<tr id='ft3.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_collect: 收藏用户分享文件</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.1')" >
        error</a>

    <div id='div_ft3.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\collect_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>create_sta.Create</td>
    <td>4</td>
    <td>0</td>
    <td>0</td>
    <td>4</td>
    <td><a href="javascript:showClassDetail('c4',4)">Detail</a></td>
</tr>

<tr id='ft4.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_create: 新建文件夹确认按钮确认</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.1')" >
        error</a>

    <div id='div_ft4.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\create_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_create2: 新建文件夹回车确认</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.2')" >
        error</a>

    <div id='div_ft4.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\create_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_create3: 新建文件夹取消按钮取消</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.3')" >
        error</a>

    <div id='div_ft4.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.3: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\create_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_create4: 新建文件夹ESC取消</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.4')" >
        error</a>

    <div id='div_ft4.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.4: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\create_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>delete2_sta.Delete2</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c5',2)">Detail</a></td>
</tr>

<tr id='ft5.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_delfile: 永久删除单个文件</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.1')" >
        error</a>

    <div id='div_ft5.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\delete2_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft5.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_delfiles: 永久删除单个文件夹</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.2')" >
        error</a>

    <div id='div_ft5.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\delete2_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>delete_sta.Delete</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c6',2)">Detail</a></td>
</tr>

<tr id='ft6.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_delete: 删除文件，文件夹到回收站</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.1')" >
        error</a>

    <div id='div_ft6.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft6.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\delete_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft6.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_empty: 一键清空回收站</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.2')" >
        error</a>

    <div id='div_ft6.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft6.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\delete_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>hotcollection_sta.Collection</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c7',1)">Detail</a></td>
</tr>

<tr id='ft7.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_heji: 收藏热门合集</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.1')" >
        error</a>

    <div id='div_ft7.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft7.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\hotcollection_sta.py", line 23, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>hotmovie_sta.Movie</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c8',1)">Detail</a></td>
</tr>

<tr id='ft8.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_dianying: 收藏热门电影</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.1')" >
        error</a>

    <div id='div_ft8.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft8.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\hotmovie_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>move_sta.Move</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c9',1)">Detail</a></td>
</tr>

<tr id='ft9.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_file: 移动文件到文件夹</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.1')" >
        error</a>

    <div id='div_ft9.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft9.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\move_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>onesetp_sta.OneSetp</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c10',2)">Detail</a></td>
</tr>

<tr id='ft10.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_intellect_rename: 智能重命名</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft10.1')" >
        error</a>

    <div id='div_ft10.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft10.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft10.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\onesetp_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft10.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_repeat: 删除重复影片</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft10.2')" >
        error</a>

    <div id='div_ft10.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft10.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft10.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\onesetp_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>openurl_sta.Openurl</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c11',1)">Detail</a></td>
</tr>

<tr id='ft11.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_add: 打开文件公开链接，收藏文件到私有云</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft11.1')" >
        error</a>

    <div id='div_ft11.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft11.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft11.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\openurl_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_aalogin_sta.OPlogin</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c12',1)">Detail</a></td>
</tr>

<tr id='ft12.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_oplogin: 用户登陆私有去公开页</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft12.1')" >
        error</a>

    <div id='div_ft12.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft12.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft12.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_aalogin_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_add_sta.OPadd</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c13',1)">Detail</a></td>
</tr>

<tr id='ft13.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_opadd: 添加链接文件到我的私有云</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft13.1')" >
        error</a>

    <div id='div_ft13.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft13.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft13.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_add_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_collect_sta.OPgather</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c14',1)">Detail</a></td>
</tr>

<tr id='ft14.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_gather: 收藏用户分享到我的私有云</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft14.1')" >
        error</a>

    <div id='div_ft14.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft14.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft14.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_collect_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_gather_sta.OPgather</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c15',1)">Detail</a></td>
</tr>

<tr id='ft15.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_gather: 收藏热门合集到我的私有云</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft15.1')" >
        error</a>

    <div id='div_ft15.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft15.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft15.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_gather_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_move_sta.OPmove</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c16',1)">Detail</a></td>
</tr>

<tr id='ft16.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_move: 收藏热门电影到我的私有云</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft16.1')" >
        error</a>

    <div id='div_ft16.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft16.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft16.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_move_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>op_usershare_sta.Usershare</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c17',1)">Detail</a></td>
</tr>

<tr id='ft17.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_usershare: 用户分享列表操作</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft17.1')" >
        error</a>

    <div id='div_ft17.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft17.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft17.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\op_usershare_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>rename_sta.Rename</td>
    <td>4</td>
    <td>0</td>
    <td>0</td>
    <td>4</td>
    <td><a href="javascript:showClassDetail('c18',4)">Detail</a></td>
</tr>

<tr id='ft18.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_rename: 文件/文件重命名回车确认</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft18.1')" >
        error</a>

    <div id='div_ft18.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft18.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft18.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\rename_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft18.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_rename2: 文件/文件重命名确认按钮确认</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft18.2')" >
        error</a>

    <div id='div_ft18.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft18.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft18.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\rename_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft18.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_rename3: Esc取消文件/文件重命名</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft18.3')" >
        error</a>

    <div id='div_ft18.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft18.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft18.3: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\rename_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft18.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_rename4: 取消按钮文件/文件重命名</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft18.4')" >
        error</a>

    <div id='div_ft18.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft18.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft18.4: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\rename_sta.py", line 18, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>restore_sta.Restore</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c19',2)">Detail</a></td>
</tr>

<tr id='ft19.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_del: 全选文件/文件夹删除到回收站</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft19.1')" >
        error</a>

    <div id='div_ft19.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft19.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft19.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\restore_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft19.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_restore: 回收站文件还原</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft19.2')" >
        error</a>

    <div id='div_ft19.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft19.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft19.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\restore_sta.py", line 19, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>share_sta.Share</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c20',2)">Detail</a></td>
</tr>

<tr id='ft20.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_gather: 分享合集</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft20.1')" >
        error</a>

    <div id='div_ft20.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft20.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft20.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\share_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft20.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_share: 分享文件</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft20.2')" >
        error</a>

    <div id='div_ft20.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft20.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft20.2: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\share_sta.py", line 17, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>usershare_sta.Usershare</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c21',1)">Detail</a></td>
</tr>

<tr id='ft21.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_usershare: 用户分享列表操作（收起，展开，换一组）</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft21.1')" >
        error</a>

    <div id='div_ft21.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft21.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft21.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\usershare_sta.py", line 23, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>xplain_sta.Xplain</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c22',1)">Detail</a></td>
</tr>

<tr id='ft22.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_xplain: 使用私有云引导说明</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft22.1')" >
        error</a>

    <div id='div_ft22.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft22.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft22.1: Traceback (most recent call last):
  File "E:\selenium_test_case\webcloud\xplain_sta.py", line 20, in setUp
    self.driver = login.browser(self)
  File "E:\selenium_test_case\webcloud\public\login.py", line 23, in browser
    'javascriptEnabled': True,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 71, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 113, in start_session
    'desiredCapabilities': desired_capabilities,
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 162, in execute
    response = self.command_executor.execute(driver_command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 349, in execute
    return self._request(url, method=command_info[0], data=data)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 410, in _request
    resp = opener.open(request)
  File "C:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "C:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: &lt;urlopen error [Errno 10061] &gt;


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>Total</td>
    <td>34</td>
    <td>0</td>
    <td>0</td>
    <td>34</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</body>
</html>
